ibis/impala/compiler.py

```diff
@@ -176,7 +168,7 @@ class ImpalaExprTranslator(BaseExprTranslator):
 
 
 class ImpalaDialect(comp.Dialect):
-    translator = ImpalaExprTranslator
+    translator = BaseExprTranslator
```

Comment:
Shouldn't this be subclassing `BaseDialect`? And if you do that, I don't think you need to set the translator, since it would be set to `BaseExprTranslator`.

And if we do that, and `ImpalaDialect` is not used anywhere else, you can probably remove this class at all, and set `dialect = BaseDialect` below.

Reply:
Dialect was used a good amount within impala so couldnt just replace it.  Let me know if you think otherwise